### PR TITLE
Fix GRUB reference in CosmosPropertyPage.xaml

### PR DESCRIPTION
**Pull request by MEMESCOEP**
I've changed "GRUB" to "Limine" because we aren't using grub anymore.

### CODE REVIEW

**Comment by Kiirx** [comment]
maybe instead we should let the person choose between grub and limine

**Comment by Kiirx** [comment]
then instead modify this not to limine but only "bootloader" to make a more general thing

File: source/Cosmos.VS.ProjectSystem/BuildSystem/Rules/CosmosPropertyPage.xaml [hunk 1]
```diff
@@ -85,7 +85,7 @@
     </EnumProperty>
 
     <StringProperty Name="Timeout"
-                    DisplayName="GRUB Menu Timeout"
+                    DisplayName="Limine Menu Timeout"
```
Comment (Kiirx) [inline source/Cosmos.VS.ProjectSystem/BuildSystem/Rules/CosmosPropertyPage.xaml:88]: Display="Bootloader Menu Timeout" instead

**Review by Kiirx — CHANGES_REQUESTED** [review]

**Comment by MEMESCOEP** [comment]
> then instead modify this not to limine but only "bootloader" to make a more general thing

Good Idea, I'll get on that now

**Review by Kiirx — APPROVED** [review]
Yep good

**Pull request merged**